docs/topics/Permissions.md

```diff
@@ -33,15 +33,15 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | MANAGE_CHANNELS \*            | `0x0000000000000010` `(1 << 4)`  | Allows management and editing of channels                                                                                                           | T, V, S      |
-| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management and editing of the guild                                                                                                          |              |
+| MANAGE_GUILD \*               | `0x0000000000000020` `(1 << 5)`  | Allows management of the server name, region, and adding bots to the server                                                                          |              |
 | ADD_REACTIONS                 | `0x0000000000000040` `(1 << 6)`  | Allows for the addition of reactions to messages                                                                                                    | T            |
 | VIEW_AUDIT_LOG                | `0x0000000000000080` `(1 << 7)`  | Allows for viewing of audit logs                                                                                                                    |              |
 | PRIORITY_SPEAKER              | `0x0000000000000100` `(1 << 8)`  | Allows for using priority speaker in a voice channel                                                                                                | V            |
 | STREAM                        | `0x0000000000000200` `(1 << 9)`  | Allows the user to go live                                                                                                                          | V            |
 | VIEW_CHANNEL                  | `0x0000000000000400` `(1 << 10)` | Allows guild members to view a channel, which includes reading messages in text channels                                                            | T, V, S      |
 | SEND_MESSAGES                 | `0x0000000000000800` `(1 << 11)` | Allows for sending messages in a channel (does not allow sending messages in threads)                                                               | T            |
 | SEND_TTS_MESSAGES             | `0x0000000000001000` `(1 << 12)` | Allows for sending of `/tts` messages                                                                                                               | T            |
-| MANAGE_MESSAGES \*            | `0x0000000000002000` `(1 << 13)` | Allows for deletion of other users messages                                                                                                         | T            |
+| MANAGE_MESSAGES \*            | `0x0000000000002000` `(1 << 13)` | Allows for deletion of or pinning other users messages                                                                                             | T            |
```

Comment:
```suggestion
| MANAGE_MESSAGES \*            | `0x0000000000002000` `(1 << 13)` | Allows for deletion of other users' messages and pinning messages                                                                                   | T            |
```

Reply:
it also allows bypassing slowmode and publishing all messages in news channels

Reply:
and removing reactions

Reply:
yeah... maybe this should just say "managing messages"

Reply:
> yeah... maybe this should just say "managing messages"

I would rather be more specific then that, since someone may be looking in these documents for specific permissions to request for a specific task (such as deleting reactions) and I would like that to be supported as fully as is reasonably possible in the documentation.